### PR TITLE
Sync with nsls2forge feedstock

### PR DESCRIPTION
**Pull request by mrakitin**
This PR is to sync with https://github.com/nsls-ii-forge/pyxrf-feedstock. I added @dmgav (current lead dev for the project) and myself as maintainers.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mrakitin** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,80 +1,92 @@
...
     - distributed
+    - h5py >=2.9.0
+    - ipywidgets
     - jsonschema
-    - h5py
-    - lmfit
-    - matplotlib
+    - jupyter
+    - lmfit >=0.8.3
+    - matplotlib <=3.2.2
```
Comment (tschoonj) [inline recipe/meta.yaml:34]: Why is this upper constraint on matplotlib necessary?

Comment (mrakitin) [reply recipe/meta.yaml:34]: @dmgav discovered some segfaults with 3.3.
@dmgav, @tacaswell, could you please provide more details?

Comment (dmgav) [reply recipe/meta.yaml:34]: PyXRF segfaults with MPL 3.3. We are investigating the issue. It is better to put upper limit than instruct users to downgrade manually.

**Review by tschoonj — APPROVED** [review]

**Comment by tschoonj** [comment]
Ready for a merge?

**Comment by mrakitin** [comment]
Yes please, @tschoonj. Thanks for your thorough review!

**Comment by mrakitin** [comment]
Thanks, @tschoonj!

**Pull request merged**